### PR TITLE
Initial build 1.15.1

### PR DESCRIPTION
**Pull request by ViridianMelody**
[Upstream repo](https://github.com/smarie/python-makefun)

Target channel: Snowflake

`great-expectations` <- `makefun`

Minimal adjustments from conda-forge.

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,18 +11,20 @@ source:
...
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - pytest-runner
-    - python >=3.5
+    - pytest-runner 6.0.0
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```

Can you try without pytest-runner? I see it listed in setup.cfg, but not in pyproject.toml.

**Review by cbouss — APPROVED** [review]
LGTM, if it builds without pytest-runner, even better!

**Pull request merged**